Eastern-Research-Group/mywaterway
# Feature/492 save map pdf

## Related Issues:
* [HMW-492](https://jira.epa.gov/browse/HMW-492)

## Main Changes:
* Added a ESRI's print widget to the map.
  * I tested using EPA's print service (https://gispub.epa.gov/arcgis/rest/services/Utilities/PrintingTools/GPServer/Export%20Web%20Map%20Task), but each time a _504 Gateway Time-out_ was returned. Per a previous discussion, using ESRI's default print service should not be an issue unless it is heavily abused.

## Steps To Test:
1. Navigate to any page with a map.
2. Open the print/download widget.
3. Test the various options, and confirm the "Layout" and "Map only" options export correctly.
